example/_config.yml

```diff
@@ -8,6 +8,9 @@ description: > # this means to ignore newlines until "baseurl:"
 baseurl: "/minima"
 twitter_username: jekyllrb
 github_username:  jekyll
+# if you want to use disqus as comment system, be aware that it only works on production
+disqus:
+  shortname: your_disqus_shortname
```

Comment:
> Should this be commented out by default, such that we don't have a bunch of sites, which by default, have a broken comment system?

Yes, lets comment this out by default. Also include a link to disqus registration page.

> Couldn't we limit the output to production via liquid..

That's how [it is](https://github.com/jekyll/minima/blob/master/_includes/disqus_comments.html#L1) right now. And its also dependent on page-wise configuration